src/operators/IntegralCurve/QvisIntegralCurveWindow.C

```diff
@@ -741,7 +741,7 @@ QvisIntegralCurveWindow::CreateAppearanceTab(QWidget *pageAppearance)
 
     dataValueComboBox = new QComboBox(dataGroup);
     dataValueComboBox->addItem(tr("Solid Color"), IntegralCurveAttributes::Solid);
-    dataValueComboBox->addItem(tr("Random Color"), IntegralCurveAttributes::SeedPointID);
+    dataValueComboBox->addItem(tr("Seed Point ID"), IntegralCurveAttributes::SeedPointID);
```

Comment:
Please have @rusu24edward review this change as it reverts a change he made last year which I think was for clarity of interface based upon input from one or more users.

Also, you may need to update the associated docs (including any screen shots if there are some) so they are consistent with with this interface (GUI) change.

Reply:
Was about to do that as there may be some documentation that will need to be updated.

@rusu24edward - I reverted the "Random" back to "Seed Point ID" because users often do analysis and want to know which how curves interact based on their ID.  And the random is really just using the seed id which is not  randomly assigned.
 
Also the variable name in the plot was still "Seed Point ID" in IntegralCurveViewerEnginePluginInfo so the change was not really fully implemented.

Reply:
> Also the variable name in the plot was still "Seed Point ID" in IntegralCurveViewerEnginePluginInfo so the change was not really fully implemented.

@ARSanderson I think the change to use `Random` for the name of the coloring scheme used was *solely* a change to outward facing GUI. Under the covers, that mapped to `Seed Point ID` and so IMHO no *more* implementation was necessary.

Reply:
@markcmiller86 Correct, but in the viewer the variable was listed at "Seed Point ID" see attached image. As such, it is odd to select "Color by" to "Random" but then in the viewer to list the color variable as "Seed Point ID". Regardless selecting random and them mapping it to "something" is not random :-).

That said I can see were a user might want a random color. In that case, use std::random_shuffle, to randomize the color value using the seed id so that no two IC have the same value.

@rusu24edward - what is/was the need for adding random? If there was specific need we can add correctly add a random assignment before this PR is merged. 

![Screen Shot 2021-04-29 at 2 27 12 PM](https://user-images.githubusercontent.com/26089249/116631820-f6336180-a912-11eb-8ff4-73a5af4bb6a9.png)

Reply:
Thanks @ARSanderson! This all looks good to me!